### PR TITLE
ign -> gz : Release Tools Migrate Internal Vars

### PR DESCRIPTION
**Pull request by methylDragon**
Part of: https://github.com/gazebo-tooling/release-tools/pull/746

### CODE REVIEW

File: jenkins-scripts/lib/dependencies_archive.sh
Comment (chapulina) [inline jenkins-scripts/lib/dependencies_archive.sh:552]: I think this needs to change, see how it's used: https://github.com/gazebo-tooling/release-tools/commit/eb1e2dd092101af977d7f571be6306a075fe46a8

```suggestion
GZ_PHYSICS_DART_FROM_PKGS="true"
```

**Review by chapulina — CHANGES_REQUESTED** [review]

File: jenkins-scripts/ign_utils-default-devel-windows-amd64.bat
Comment (chapulina) [reply jenkins-scripts/ign_utils-default-devel-windows-amd64.bat:12]: I see these changes are also in #752, so we can revert them here

**Comment by methylDragon** [comment]
Ah, I remember why there are many duplications here. I had a lot of conflicts if I didn't base the branch off the project name migration. I think we should merge the name migration first, then rebase this! Otherwise we'll have a lot of conflicts  for the project name migration.

File: jenkins-scripts/lib/dependencies_archive.sh
Comment (methylDragon) [reply jenkins-scripts/lib/dependencies_archive.sh:392]: Updated!

**Comment by j-rivero** [comment]
> Ah, I remember why there are many duplications here. I had a lot of conflicts if I didn't base the branch off the project name migration. I think we should merge the name migration first, then rebase this! Otherwise we'll have a lot of conflicts for the project name migration.

Is there anything missing from the name migration that is blocking this PR? I'm happy to help to unblock this one. Thanks.

**Comment by methylDragon** [comment]
> > Ah, I remember why there are many duplications here. I had a lot of conflicts if I didn't base the branch off the project name migration. I think we should merge the name migration first, then rebase this! Otherwise we'll have a lot of conflicts for the project name migration.
> 
> Is there anything missing from the name migration that is blocking this PR? I'm happy to help to unblock this one. Thanks.

I think there aren't any blockers here, we just weren't sure about merging before feature freeze since it could've broken stuff :o
Let me resolve the conflicts and then you can give a look-see to see if what I've changed makes sense?

File: jenkins-scripts/ign_common-default-devel-windows-amd64.bat
Comment (j-rivero) [inline jenkins-scripts/ign_common-default-devel-windows-amd64.bat:7]: Was this removal an error or there is a reason to remove it in this PR?

File: jenkins-scripts/ign_gazebo-default-devel-windows-amd64.bat
Comment (j-rivero) [inline jenkins-scripts/ign_gazebo-default-devel-windows-amd64.bat:8]: same assimp removal here

File: jenkins-scripts/ign_common-default-devel-windows-amd64.bat
Comment (methylDragon) [reply jenkins-scripts/ign_common-default-devel-windows-amd64.bat:7]: Humm, this might've happened when I rebased, I think it's an error, I'll revert the change

File: jenkins-scripts/ign_gazebo-default-devel-windows-amd64.bat
Comment (methylDragon) [reply jenkins-scripts/ign_gazebo-default-devel-windows-amd64.bat:8]: https://github.com/gazebo-tooling/release-tools/pull/753/commits/d513c8d8cbdd57af2be151f14154d6233dddd16d

File: jenkins-scripts/ign_common-default-devel-windows-amd64.bat
Comment (methylDragon) [reply jenkins-scripts/ign_common-default-devel-windows-amd64.bat:7]: https://github.com/gazebo-tooling/release-tools/pull/753/commits/d513c8d8cbdd57af2be151f14154d6233dddd16d

**Comment by scpeters** [comment]
testing windows script changes:

* [Build Status](https://build.osrfoundation.org/job/ign_math-ign-6-win/) https://build.osrfoundation.org/job/ign_math-ign-6-win/46/
* [Build Status](https://build.osrfoundation.org/job/ign_math-gz-7-win/2/) https://build.osrfoundation.org/job/ign_math-gz-7-win/2/

**Review by scpeters — CHANGES_REQUESTED** [review]

File: jenkins-scripts/ign_common-default-devel-windows-amd64.bat
Comment (j-rivero) [reply jenkins-scripts/ign_common-default-devel-windows-amd64.bat:7]: good catch! [6138403](https://github.com/gazebo-tooling/release-tools/pull/753/commits/61384030c3e9a567e801e64b6be92b47038e9095)

**Comment by scpeters** [comment]
I was going to start a test build of ign-gazebo6 on windows, but that has been broken since https://github.com/gazebosim/gz-sim/pull/1626

**Review by scpeters — APPROVED** [review]

**Pull request merged**
